src/ansys/fluent/core/session_pure_meshing.py

```diff
@@ -238,3 +237,15 @@ def transfer_mesh_to_solvers(
             clean_up_mesh_file,
             overwrite_previous,
         )
+
+    def enable_beta_features(self):
+        """Enable access to Fluent beta-features"""
+        if not self._app_utilities.is_beta_enabled():
+            self.tui.beta_feature_access("yes", "yes")
+
+    def __dir__(self):
+        if self._fluent_connection is None:
+            return ["is_active"]
```

Comment:
@prmukherj The strategy of what to do when detached is a core decision, not for concrete leaf classes.